ChangeLog

```diff
@@ -37,6 +37,10 @@ Release date: TBA
 
   Closes #5281
 
+* Fixed false negative ``unpacking-non-sequence`` when value is an empty list.
+
+ Closes #5707
+
```

Comment:
```suggestion

* Fix false positive - Allow unpacking of ``self`` in a subclass of ``typing.NamedTuple``.

  Closes #5312
  Closes #5671
```

Reply:
Updated. Note the 5671 should never have been there; removed.